<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Записнaя книжка</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="load">
           <property name="text">
            <string>Загрузить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="save">
           <property name="text">
            <string>Сохранить</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QCheckBox" name="name">
             <property name="text">
              <string>Имя:</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="second_name">
             <property name="text">
              <string>Фамилия:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="email">
             <property name="text">
              <string>E-mail:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="telephone">
             <property name="text">
              <string>Телефон:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="nameE">
             <property name="maxLength">
              <number>32764</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="second_nameE"/>
           </item>
           <item>
            <widget class="QLineEdit" name="emailE"/>
           </item>
           <item>
            <widget class="QLineEdit" name="telephoneE"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QRadioButton" name="all_gal">
           <property name="text">
            <string>Выделить галочки</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="no_gal">
           <property name="text">
            <string>Снять галочки</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="clear">
           <property name="text">
            <string>Очистить поля</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="view">
           <property name="text">
            <string>Вывети</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>Имя</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Фамилия</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>E-mail</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Телефон</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
